packages/cells/src/widget.ts

```diff
@@ -864,6 +864,7 @@ export class MarkdownCell extends Cell {
     this._updateRenderedInput().then(() => {
       this._ready.resolve(void 0);
     });
+    this.renderInput(this._renderer);
```

Comment:
What does this change accomplish?

Reply:
In order to scroll into view the fragment we have to wait cell output is inserted into DOM.
However `_updateRenderedInput()` just generates the output without injecting it into DOM.
So, without that change, the `ready` promise resolves too early for our needs.